apply-ParagraphStandards.py

```diff
@@ -0,0 +1,382 @@
+import glob
+import json
+import yaml
+import getopt
+import sys
+from pprint import pprint
+from dateutil import parser
+
+
+def usage():
+    """Print the standard usage for calling the script.
+    Outputs valid options.
+
+    Arguments: none
+    """
+
+    print('Usage:')
+    print("\t -m                       : modify source files")
+    print("\t -t:{test}, --test {test} : run specific tests, or All (default)")
+    print("\t -h, or --help            : display help")
+    print("\t Test list:")
+    print("\t\t All")
+    print("\t\t MDTimeCheck")
+    print("\t\t CodeBlockFix")
+
+
+def logline(logdata):
+    """Standardize the log output making it easier to scan through
+
+    Arguments:
+    logdata   -- array of the following data
+                 notebook_id, paragraph #, paragraph title, log text
+    """
+
+    print("{:<10} pg#{:<2} {:<40} {:<25}".format(logdata[0],
+          logdata[1], logdata[2][:40], logdata[3]))
+
+
+def get_exceptiondetail(notebook_id, paragraph_id,
+                        field_name, exceptions_list):
```

Comment:
I was afraid it might be too close to a potential keyword (exception) which was why I added the list.  Easy enough to make the change if my fear is unfounded.